src/test/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/htmlTags/expectedHtmlVoidElementEmbedAst.txt

```diff
@@ -734,7 +734,7 @@ JAVADOC -> JAVADOC [0:0]
 |           |--ATTRIBUTE -> ATTRIBUTE [39:2]
 |           |   |--HTML_TAG_NAME -> attr2 [39:2]
 |           |   |--EQUALS -> = [39:7]
-|           |   `--ATTR_VALUE -> "someotherting" [39:8]
+|           |   `--ATTR_VALUE -> "someotherthing" [39:8]
```

Comment:
Apparently, changes to this file / its master involve recalculating subsequent lines. I'm trying to figure out what the test wants to do.

Reply:
Positions of ``--END -> >` and ``--SLASH_END -> />` have to be increased.
You changed the exact character position of `>` and `/>` for the line by increasing/decreasing the length of `someotherting`.

Reply:
I take it that you're ok with this change in general?

I'll try to do the adjustments tonight...

Reply:
I am generally fine with spelling fixes, but be careful changing input files.
Some tests could require could have an invalid/weird name for a reason. This should be a very rare occurrence though.